plugins/MantisCoreFormatting/lang/strings_english.txt

```diff
@@ -56,3 +56,16 @@ $s_plugin_format_process_markdown = 'Markdown Processing';
 $s_plugin_format_process_markdown_info = 'Styling with Markdown support, text formatting syntax designed so that it can be converted to HTML.';
 $s_plugin_format_enabled = 'On';
 $s_plugin_format_disabled = 'Off';
+$s_plugin_format_syntax_highlighting = 'Syntax highlighting for code blocks';
+$s_plugin_format_syntax_highlighting_info = 'Syntax highlighting works only if "Markdown processing" is active.';
+$s_plugin_format_syntax_highlighting_theme = 'Appearance/Theme of the code blocks';
+$s_plugin_format_syntax_highlighting_plugins = 'Extensions for the code blocks';
```

Comment:
Suggest changing above 2 strings to
- `Code blocks theme`
- `Code blocks options`